src/interchange/src/avro.rs

```diff
@@ -299,9 +300,11 @@ impl<'a> AvroDecode for DebeziumSourceDecoder<'a> {
                 return Err(DecodeError::Custom("no file".to_string()).into());
             }
             RowCoordinates::MySql { pos, row }
-        } else {
+        } else if pg_any {
             let lsn = lsn.ok_or_else(|| DecodeError::Custom("no lsn".to_string()))? as usize;
             RowCoordinates::Postgres { lsn }
+        } else {
+            RowCoordinates::Unknown
```

Comment:
could we add an info- or possibly warn-level log the first time that this happens that we're not sure that we will handle duplicate messages correctly? We're not testing this code path in-house so it's hard to say that it's anything other than alpha at best.

Reply:
We absolutely should; good catch.